packages/admin/cms-admin/src/generator/utils/find-root-blocks.ts

```diff
@@ -0,0 +1,61 @@
+import { existsSync } from "fs";
```

Comment:
File names in Admin code should be camelCase, you may change that for all generator files